File: Models/Feedback.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CustomerFeedback.Models
{
    public class Feedback
    {
        public int Id { get; set; }
        public string Title { get; set; } = null!;
        public string Description { get; set; } = null!;
        public int Rating { get; set; }
        public DateTime DateReviewed { get; set; }
        public string UserId { get; set; } = null!;

        [ForeignKey(nameof(UserId))]
        public virtual AppUser User { get; set; } = null!;
    }
}

File: ClientApp/src/models/feedback.ts
export interface Feedback {
  id: string;
  title: string;
  description: string;
  rating: number;
  dateReviewed: string;
  userName: string;
}

export interface FeedbackFormValues {
  title: string;
  description: string;
  rating: number;
  dateReviewed: string;
}
